docs/api/basic.rst

```diff
@@ -0,0 +1,298 @@
+Basic Usage
+===========
+
+We are small package, so our API is not so big. There are only few classes, which are suggested for a basic usage.
+
+
+Request Classes
+---------------
+
+These are classes, that you use to send a request to server.
+
+
+.. autoclass:: mcstatus.server.MCServer
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+.. autoclass:: mcstatus.server.JavaServer
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+.. autoclass:: mcstatus.server.BedrockServer
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+
+Response Objects
+----------------
+
+But those were request classes, how about response ones?
+
+For Java Server
+***************
+
+.. class:: PingResponse
+    :canonical: mcstatus.pinger.PingResponse
+
+    The response object for :meth:`JavaServer.status() <mcstatus.server.JavaServer.status>`.
+
+    .. class:: Players
+        :canonical: mcstatus.pinger.PingResponse.Players
+
+        Class for storing information about players on the server.
+
+        .. class:: Player
+            :canonical: mcstatus.pinger.PingResponse.Players.Player
+
+            Just a class for a player information.
+
+            .. attribute:: name
+                :type: str
+                :canonical: mcstatus.pinger.PingResponse.Players.Player.name
+
+                Name of the player.
+
+            .. attribute:: id
+                :type: str
+                :canonical: mcstatus.pinger.PingResponse.Players.Player.id
+
+                ID of the player (in `UUID <https://en.wikipedia.org/wiki/Universally_unique_identifier>`_ format).
+
+        .. attribute:: online
+            :type: int
+            :canonical: mcstatus.pinger.PingResponse.Players.online
+
+            Current number of online players.
+
+        .. attribute:: max
+            :type: int
+            :canonical: mcstatus.pinger.PingResponse.Players.max
+
+            Maximum allowed number of players.
+
+        .. attribute:: sample
```

Comment:
We might want to consider letting people know that this is what appears when you hover over the slot count on the multiplayer screen. Perhaps we could even include a screenshot of this, as this feature is often misunderstood.

Reply:
Added [here](https://github.com/py-mine/mcstatus/compare/f33d6f5e6c80269ef9072380bd2966a4ca44843e..f8b35759655c63545aab056f5beaae62637d3ba7).